integrations/access/accesslist/app.go

```diff
@@ -116,35 +115,28 @@ func (a *App) run(ctx context.Context) error {
 		return nil
 	})
 
-	remindInterval := interval.New(interval.Config{
-		Duration:      time.Hour * 3,
-		FirstDuration: utils.FullJitter(time.Second * 30),
-		Jitter:        retryutils.NewSeventhJitter(),
-		Clock:         a.clock,
-	})
-	defer remindInterval.Stop()
 	log := logger.Get(ctx)
 
 	log.Info("Access list monitor is running")
 
 	a.job.SetReady(true)
 	for {
 		select {
-		case <-remindInterval.Next():
+		case <-a.clock.After(reminderInterval):
```

Comment:
why is a static `clock.After()` better than a jittered interval with a ~30s first duration? It's easier for testing purposes? I'm afraid that if your plugin restarts every 2 hours for some reason you will never get any notification with this ticker.

Reply:
Our `interval` utility is not very testable right now, and `clock.After()` allows for a deterministic test that behaves the same every run and doesn't depend on `require.Eventually`.

Valid point about restarting every 2 hours though. I wonder if we can solve for that by doing one check manually before entering the loop..